libbeat/common/mapstr.go

```diff
@@ -11,9 +11,8 @@ import (
 // Event metadata constants. These keys are used within libbeat to identify
 // metadata stored in an event.
 const (
-	EventMetadataKey = "_event_metadata"
```

Comment:
I assume this already has become obsolete in previous refactoring?

Reply:
The symbol still has been used in the processors pipeline. This PR removes support for `_event_metadata` in the pipeline -> can finally remove EventMetadataKey